ironic.conf

```diff
@@ -20,7 +21,7 @@ deploy_logs_collect = always
 deploy_logs_local_path = /shared/log/ironic/deploy
 
 [conductor]
-automated_clean = true
+automated_clean = false
```

Comment:
This change is unrelated, but forcing it in does address a known hardware issue with IPA. This.. ideally... should be reverted once the IPA change has percolated through the release pipelines/processes.